src/debugger/configuration/resolvers/ros1/launch.ts

```diff
@@ -32,7 +34,31 @@ export class LaunchResolver implements vscode.DebugConfigurationProvider {
         if (!path.isAbsolute(config.target) || path.extname(config.target) !== ".launch") {
             throw new Error("Launch request requires an absolute path as target.");
         }
+
+        const delay = ms => new Promise(res => setTimeout(res, ms));
+
+        switch(env.ROS_VERSION.trim()) {
+            case "1": {
+                // Manage the status of the ROS core, starting one if not present
+                // The ROS core will continue to run until the VSCode window is closed
+                const core_active = rosApi.getCoreStatus();
+                if (!(await core_active).valueOf()) {
+                    rosApi.startCore();
         
+                    // Wait for the core to start
+                    while (!(await rosApi.getCoreStatus()).valueOf()) {
+                        await delay(100);
+                    }
+                }
+                break;
+            }
+            case "2": {
+                // TODO, support starting the ROS2 daemon automatically
```

Comment:
With a Todo comment, could you link it to a github issue so we don't forget to add this?

Reply:
I created #431 for the issue. I wasn't sure on the exact format you wanted to link it, so I went with `// TODO(#431)` on both relevant comments if that's alright.